README.md

```diff
@@ -1,24 +1,43 @@
-# Color Grid Game (working title)
+# Color Grid Game
 
-The color grid game allows users to generate a customizable grid consisting of differently-colored tiles. This is an exciting way to explore and enjoy colors. Users can take it a step further--they can then choose to make this grid into a puzzle game that randomizes the grid so that they can put it back together in the correct order.
+A tile-switching game allowing users to put an shuffled grid of colors in color order.
 
-[This codepen](https://codepen.io/riseerpelding/pen/qBqzwvq) contains most of the work I've done on this so far.
+## Setup
+1. `git clone git@github.com:rise-erpelding/color-grid-game.git` Clone the project
+2. `npm install` to install dependencies
+3. `npm start` to run the app locally
+4. `npm test` to run tests
+5. `npm run lint` to run linter
+6. `npx stryker run` to run mutation testing
```

Comment:
These steps might do better under a "Contributing" heading, since they're not part of the setup.